README.md

```diff
@@ -101,10 +101,10 @@ Additional information is available in the repository's [Wiki](https://github.co
 The solutions-geoprocessing-toolbox repo is now supporting toolboxes for both ArcMap/ArcCatalog/ArcGlobe/ArcScene (collectively called ArcGIS for Desktop) and also ArcGIS Pro. Toolboxes that are modified in ArcGIS Pro are not
 backwards compatible with other ArcGIS Desktop applications (ArcMap), so most toolboxes are duplicated for one or the other. The naming of these toolboxes is as follows:
 
-* Toolboxes that are for ArcGIS Desktop 10.3 will include *_10.3* after the toolbox name. For example: **Visibility and Range Tools_10.3.tbx**
+* Toolboxes that are for ArcGIS Desktop 10.4 will include *_10.4* after the toolbox name. For example: **Visibility and Range Tools_10.4.tbx**
```

Comment:
Did the tbx format change at 10.4 (i.e. can a 10.4 tbx not be opened on 10.3 - I know I didn't have any problem viewing on 10.3)? If not, then there probably was not a reason to rename these files and the directions could just state *10.3 or later*